app/src/androidTest/java/org/mozilla/fenix/ui/SitePermissionsTest.kt

```diff
@@ -94,29 +117,13 @@ class SitePermissionsTest {
         }
     }
 
-    @SmokeTest
-    @Test
-    fun blockAppUsingAudioVideoTest() {
-        navigationToolbar {
-        }.enterURLAndEnterToBrowser(testPage.toUri()) {
-        }.clickStartAudioVideoButton {
-            // allow app to record video
-            clickAppPermissionButton(false)
-            // allow app to record audio
-            clickAppPermissionButton(false)
```

Comment:
can we keep this test for full coverage by using the clickDenyAppPermissionButton() method? I see there's a rule to grant camera and mic permissions by default, so maybe this is feasible only in a separate test class?

Reply:
Note: Re-visit after landing as is

Reply:
Tracking in: https://github.com/mozilla-mobile/fenix/issues/23298